torch/onnx/_internal/fx/_type_utils.py

```diff
@@ -0,0 +1,392 @@
+"""Utilities for converting and operating on ONNX, JIT and torch types."""
+from __future__ import annotations
+
+import enum
+import typing
+from typing import Any, Dict, List, Literal, Optional, Tuple, Union
+
+import torch
+from torch._C import _onnx as _C_onnx
+from torch.onnx import errors
+from torch.onnx._internal import _beartype
+
+if typing.TYPE_CHECKING:
+    # Hack to help mypy to recognize torch._C.Value
+    from torch import _C  # noqa: F401
+
+ScalarName = Literal[
+    "Byte",
```

Comment:
These seem obsolete to our new exporter?

Reply:
Trimmed